### PR TITLE
Remove capital letters from local variables

### LINKED ISSUE #364: remove capital letters in local variable/class variable names
## Description: 
remove capital letters in local variable/class variable names in:-
- analysis.modules.external_calls
- analysis.modules.unchecked_retval
- ether.asm
- ether.evm
- leveldb.client
- leveldb.eth_db
- leveldb.state
- mythril.py
- rpc.base_client
- support.truffle
- tests.ethcontract_test

### PR DESCRIPTION
**Pull request by rbval**
Fixes #364 . Mainly deleted camelCased local variables. If there's some other fixes to be done, I can change them in this PR or in another.

### CODE REVIEW

**Comment by norhh** [comment]
Great job, thanks for the PR. There are some few changes required.

File: mythril/leveldb/client.py [hunk 1]
```diff
@@ -11,21 +11,21 @@
...
-blockHashPrefix = b'H'      # blockHashPrefix + hash -> num (uint64 big endian)
-blockReceiptsPrefix = b'r'  # blockReceiptsPrefix + num (uint64 big endian) + hash -> block receipts
+header_prefix = b'h'         # header_prefix + num (uint64 big endian) + hash -> header
+body_prefix = b'b'           # body_prefix + num (uint64 big endian) + hash -> block body
+num_suffix = b'n'            # header_prefix + num (uint64 big endian) + num_suffix -> hash
+block_hash_prefix = b'H'      # block_hash_prefix + hash -> num (uint64 big endian)
+block_receipts_prefix = b'r'  # block_receipts_prefix + num (uint64 big endian) + hash -> block receipts
 # known geth keys
-headHeaderKey = b'LastBlock'  # head (latest) header hash
+head_header_key = b'last_block'  # head (latest) header hash
```
Comment (norhh) [inline mythril/leveldb/client.py]: 'LastBlock' should not be changed, That's how things are in ethereum's DB.

File: mythril/leveldb/client.py [hunk 2]
```diff
@@ -11,21 +11,21 @@
...
 # custom prefixes
-addressPrefix = b'AM'       # addressPrefix + hash -> address
+address_prefix = b'AM'       # address_prefix + hash -> address
 # custom keys
-addressMappingHeadKey = b'accountMapping'  # head (latest) number of indexed block
-headHeaderKey = b'LastBlock'  # head (latest) header hash
+address_mapping_head_key = b'account_mapping'  # head (latest) number of indexed block
+head_header_key = b'last_block'  # head (latest) header hash
```
Comment (norhh) [inline mythril/leveldb/client.py]: Same as above. 'LastBlock' should not be changed, That's how things are in ethereum's DB.

File: mythril/leveldb/client.py [hunk 3]
```diff
@@ -11,21 +11,21 @@
...
 # known geth keys
-headHeaderKey = b'LastBlock'  # head (latest) header hash
+head_header_key = b'last_block'  # head (latest) header hash
 # custom prefixes
-addressPrefix = b'AM'       # addressPrefix + hash -> address
+address_prefix = b'AM'       # address_prefix + hash -> address
 # custom keys
-addressMappingHeadKey = b'accountMapping'  # head (latest) number of indexed block
-headHeaderKey = b'LastBlock'  # head (latest) header hash
+address_mapping_head_key = b'account_mapping'  # head (latest) number of indexed block
```
Comment (norhh) [inline mythril/leveldb/client.py]: can you revert this back too due to the reasons mentioned above

**Comment by rbval** [comment]
Byte strings reverted @norhh

**Comment by norhh** [comment]
great job.

**Pull request merged**